### PR TITLE
Increase frequency getLastIndexedBlock runs

### PR DESCRIPTION
**Pull request by markdavison**

### CODE REVIEW

**Review by jdomingos — COMMENTED** [review]
I think we are calling the fn at most twice with that function. Left a comment about it :)

File: packages/indexer-serverless/src/functions/last-indexed-block/handler.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import mongoose from 'mongoose';
-import { repository } from 'indexer-utils';
+import { repository, utils } from 'indexer-utils';
 
 export default async function lastIndexedBlock() {
   await mongoose.connect(process.env.MONGO_URI!);
-  await repository.lastIndexedBlock.calculate();
+  await utils.callXTimesOverYSeconds(5, 50, () =>
+    repository.lastIndexedBlock.calculate(),
```
[the remark this answers is not in the document]
Comment (markdavison) [reply packages/indexer-serverless/src/functions/last-indexed-block/handler.ts:7]: Good point

File: packages/indexer-utils/src/utils/call-x-times-over-y-seconds.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export default (iterations: number, maxTime: number, fn: () => any) =>
+  new Promise((resolve) => {
+    const interval = (maxTime * 1000) / iterations;
+
+    const callFn = () => {
+      fn();
+      if (--iterations <= 0) {
+        return resolve(null);
+      }
+      setTimeout(callFn, interval);
```
[the remark this answers is not in the document]
Comment (markdavison) [reply packages/indexer-utils/src/utils/call-x-times-over-y-seconds.ts:10]: Pretty sure it works I ran the loop with some console logs and checked that it invoked at the correct intervals

[the remark this answers is not in the document]
Comment (jdomingos) [reply packages/indexer-utils/src/utils/call-x-times-over-y-seconds.ts:10]: Sorry, my bad, my eyes got confused with fn and callFn 😵‍💫

Ignore the comment

**Review by jdomingos — APPROVED** [review]

**Pull request merged**